helm/cluster-vsphere/templates/ipam/assign-ip-pre-install-job.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if eq (include "isIpamEnabled" $) "true" -}}
+{{- if or (eq (include "isIpamEnabled" $) "true") (eq (include "isIpamSvcLoadBalancerEnabled" $) "true") -}}
```

Comment:
do you need that extra condition? Can `isIpamSvcLoadBalancerEnabled` exist without `isIpamEnabled` ?

Reply:
maybe I can rename the `isIpamEnabled` -> `isIpamControlPlaneIPEnabled`? so that it doesn't look like these options are kind of hierarchical?

Reply:
Yep that'd be a good one I think